daemon/inspect_fs_unix_fstab.ml

```diff
@@ -542,6 +542,32 @@ and resolve_dev_mapper spec default =
   (* Ignore devices that don't exist. (RHBZ#811872) *)
   | Unix.Unix_error (Unix.ENOENT, _, _) -> default
 
+and resolve_crypttab spec aug =
+  let augpath = sprintf "/files/etc/crypttab/*[target='%s']/device"
+    (Filename.basename spec) in
```

Comment:
This is a bit confusingly indented.  You could write either indent the parameter much further (so it's still on the next line, but to the right of the `sprintf`); or:
```
let augpath =
  sprintf "..." (Filename.basename spec) in
```
or even:
```
let augpath = Filename.basename spec |> sprintf "..." in
```